sys/net/crosslayer/ng_pktbuf_static/ng_pktbuf_static.c

```diff
@@ -429,6 +429,8 @@ static inline bool _too_small_hole(_unused_t *a, _unused_t *b)
 
 static inline _unused_t *_merge(_unused_t *a, _unused_t *b)
 {
+    assert(b != NULL);
```

Comment:
Then there should be an if check before the call to `_merge` in l461, since `new->next` can be `NULL`, if the `while`-loop in l445 was terminated by that case. Should be enough to add a `(new->next != NULL) &&` before the `too_small_hole(new, new->next)`.